lessons/intro/mypy/index.md

```diff
@@ -146,6 +146,50 @@ Té se to právem nelíbí, protože v definici jsme si stanovili, že funkce m
 brát jako argument řetězec a místo toho jí na posledním řádku voláme
 s celočíselným argumentem.
 
+
+### Vlastní datové typy
+
+V Pythonu často používáme i vlastní datové typy (například instance tříd), 
+nejen ty vestavěné (jako je `str`, `list` a další).
+
+```python
+class Animal:
+    name: str
```

Comment:
U tříd bych použil raději `__init__` a jméno nastavil při tvorbě objektu, čímž bychom se více přiblížili tomu, jak je to učíme v začátečnickém kurzu a nemuseli bychom tady vedle typové kontroly vysvětlovat i třídní proměnné.